diff -urNp gnome-power-manager-2.28.3.old/src/gpm-prefs-core.c gnome-power-manager-2.28.3/src/gpm-prefs-core.c
--- gnome-power-manager-2.28.3.old/src/gpm-prefs-core.c	2011-02-01 15:29:05.492597000 +0000
+++ gnome-power-manager-2.28.3/src/gpm-prefs-core.c	2011-02-01 15:28:58.356597000 +0000
@@ -379,11 +379,8 @@ gpm_prefs_setup_action_combo (GpmPrefs *
 			gtk_combo_box_append_text (GTK_COMBO_BOX (widget), _("Ask me"));
 			g_ptr_array_add (array, GINT_TO_POINTER (policy));
 		} else if (policy == GPM_ACTION_POLICY_NOTHING) {
-			/* we only add do nothing in the GUI if the user has explicitly specified this in GConf */
-			if (value == GPM_ACTION_POLICY_NOTHING) {
-				gtk_combo_box_append_text (GTK_COMBO_BOX (widget), _("Do nothing"));
-				g_ptr_array_add (array, GINT_TO_POINTER (policy));
-			}
+			gtk_combo_box_append_text (GTK_COMBO_BOX (widget), _("Do nothing"));
+			g_ptr_array_add (array, GINT_TO_POINTER (policy));
 		} else {
 			egg_warning ("Unknown action read from conf: %i", policy);
 		}
